packages/radio/src/spectrum-radio.css

```diff
@@ -484,8 +483,7 @@ governing permissions and limitations under the License.
             var(--spectrum-radio-focus-indicator-gap) * 2
     );
 }
-#input:focus-visible + #button:after,
-:host(:focus-visible) #input + #button:after {
+#input:focus-visible + #button:after {
```

Comment:
This seems to be the only component in this PR where there was also a `.focus-visible` class. @Westbrook is that something that needs to be kept or is using the pseudo-class enough?